docs/node/failures.md

```diff
@@ -0,0 +1,68 @@
+---
+id: failures
+title: Failures in Node
+sidebar_label: Failures
+---
+
+Failure classes are used by Temporal to represent failures across the different SDKs for both Activities and Workflows.
+
+Failures are serializable over network and chainable to enable root cause analysis anywhere in the failure chain. If, for example, a Node.js Workflow starts a Java Child Workflow which calls an Activity in Golang and that activity fails, the Node.js Workflow will throw a `ChildWorkflowFailure` with `cause` set to `ActivityFailure` with `cause` set to `ApplicationFailure` representing the error that occured in the Golang Activity.
+
+Outside of Workflow code, failure classes are attached to the `cause` of [`WorkflowExecutionFailedError`](https://nodejs.temporal.io/api/classes/client.workflowexecutionfailederror) which is thrown when executing a Workflow with a [`WorkflowClient`](https://nodejs.temporal.io/api/classes/client.workflowclient/) or [`WorkflowStub`](https://nodejs.temporal.io/api/interfaces/client.workflowstub/).
+
+Failures are also used to represent [cancellation](/docs/node/cancellation-scopes#cancelledfailure) of Activities and Child Workflows.
```

Comment:
after reading this I still have no idea when to use it. am i supposed to check `instanceof` errors? am I supposed to create `new ActivityFailure()`s? can you spell out top 2-3 expected use cases?

Reply:
I'll fix this